gdal/src/main/scala/geotrellis/contrib/vlm/gdal/GDALDataset.scala

```diff
@@ -42,7 +42,7 @@ case class GDALDataset(token: Long) extends AnyVal {
     val width_height = Array.ofDim[Int](2)
     if (GDALWarp.get_transform(token, dataset, numberOfAttempts, transform) <= 0 ||
       GDALWarp.get_width_height(token, dataset, numberOfAttempts, width_height) <= 0)
-      throw new Exception("get_transform or get_width_height")
+      throw new MalformedDataException("Bad tranform values read")
```

Comment:
Can you think of a better exception messages? In this case it looks like it can't construct `RasterExtent`.